dev/PushNotifications/PushNotificationsLongRunningTask/winmain.cpp

```diff
@@ -0,0 +1,7 @@
+﻿#include <windows.h>
```

Comment:
Missing header comments (Copyright and MIT License)

Same comment applies throughout